net-dialup/freeradius-client/freeradius-client-1.1.6.ebuild

```diff
@@ -17,15 +18,20 @@ DEPEND="!net-dialup/radiusclient
 	!net-dialup/radiusclient-ng"
 RDEPEND="${DEPEND}"
 
+src_prepare() {
+	epatch "${FILESDIR}/${P}-BJA-cross.diff"
+	eautoreconf
+}
+
 src_configure() {
 	econf \
 		$(use_enable scp) \
 		$(use_enable shadow) \
-		--with-secure-path || die "econf failed"
+		--with-secure-path
 }
 
 src_install() {
-	emake DESTDIR="${D}" install || die "emake install failed"
+	emake install
 
 	dodoc BUGS README* doc/ChangeLog doc/login.example doc/release-method.txt
 	newdoc doc/README README.login.example
```

Comment:
Well the dohtml was already there, do you want to me to do that as long as the eapi bump ?